Competition2018/src/main/java/competition/subsystems/climb/commands/AscendClimberCommand.java

```diff
@@ -0,0 +1,28 @@
+package competition.subsystems.climb.commands;
+
+import com.google.inject.Inject;
+import competition.subsystems.climb.ClimbSubsystem;
+import xbot.common.command.BaseCommand;
+
+public class AscendClimberCommand extends BaseCommand{
+	
+	ClimbSubsystem climb;
+	
+	@Inject
+	public AscendClimberCommand (ClimbSubsystem climb) {
+		this.climb = climb;
+		this.requires(climb);
+	}
+	
+	@Override
+	public void initialize() {
+		log.info("Initializing");
+		
```

Comment:
remove blank lines like these at the end of functions

Reply:
done